src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -20,8 +20,8 @@ type TPlaygroundSection<T extends TSocketEndpointNames> = {
   error: unknown;
 };
 
-const LoginModal = (visible) => {
-  if (visible) {
+export const LoginModal = (visible) => {
```

Comment:
Same here, we do not use this export anywhere outside this file.

Reply:
Yes we will use this for testing the component